SWLOR.Game.Server/Feature/TrapDefinition/Trap.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+
+namespace SWLOR.Game.Server.Feature
+{
+    public class PitfallTrap
+    {
+        /// <summary>
+        /// When this trap is triggered, it'll toss the players to the designated waypoint after a delay.
+        /// </summary>
+        [NWNEventHandler("pitfalltrap")]
```

Comment:
This trap type is not in your PR. Please ensure it's been added to the palette. There should be a .utt file for the trap trigger.